client/src/app/pages/applications/components/application-tags/application-tags.tsx

```diff
@@ -48,38 +50,44 @@ export const ApplicationTags: React.FC<ApplicationTagsProps> = ({
 
           Promise.all(
             tagValidResponses.map((tag) =>
-              getTagTypeById(tag?.tagType?.id || 0)
+              getTagCategoryById(tag?.category?.id || 0)
```

Comment:
so this is .category and not .tagCategory? Just a question.

Reply:
Yes the backend changed tag.tagCategory to tag.category